Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/TimeView/TimeViewPlugin.h

```diff
@@ -201,12 +206,16 @@ namespace EMStudio
         MCORE_DEFINECOMMANDCALLBACK(UpdateInterfaceCallback);
         AZStd::vector<MCore::Command::Callback*> m_commandCallbacks;
 
-        TrackDataHeaderWidget* m_trackDataHeaderWidget;
-        TrackDataWidget*    m_trackDataWidget;
-        TrackHeaderWidget*  m_trackHeaderWidget;
-        TimeInfoWidget*     m_timeInfoWidget;
-        TimeViewToolBar*    m_timeViewToolBar;
-        QWidget*            m_mainWidget;
+        TrackDataHeaderWidget*      m_trackDataHeaderWidget;
+        TrackDataWidget*            m_trackDataWidget;
+        TrackHeaderWidget*          m_trackHeaderWidget;
+        TimeInfoWidget*             m_timeInfoWidget;
+        TimeViewToolBar*            m_timeViewToolBar;
+        MotionEventPresetsWidget*   m_motionEventsPresetsWidget;
+        QWidget*                    m_mainWidget;
+        QSplitter*                  m_paneSplitter; // motion presets pane
+
+        QAction*                    m_togglePresetsView;
```

Comment:
Can you initialize these with nullptr please?

Reply:
sure thing